extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -29,3 +29,12 @@
   padding-right: 20px;
   border: solid 1px #dfdfdf;
 }
+
+amp-accordion.i-amphtml-display-locking > section:not([expanded]) > :last-child,
```

Comment:
We will need some code here for `hidden` as well, since we will no longer be able to use the `hidden` attribute.

Reply:
Added some additional CSS after removing `hidden` in latest revision.